src/components/RecordFormTitle/RecordFormTitle.jsx

```
@@ -43,34 +60,63 @@ const RecordFormTitle = ({
+  const { mermaidExploreLink, isExploreLaunchEnabledForUser } = useExploreLaunchFeature({
+    currentUser,
+  })
 
   useDocumentTitle(
     `${primaryTitle && `${primaryTitle} `}${siteName} ${transectNumber} - ${
       language.title.mermaid
     }`,
   )
 
+  const handleExploreButtonClick = () => {
+    window.open(`${mermaidExploreLink}/?sample_event_id=${projectName}`, '_blank')
```

ESLint

[error] 'projectName' is not defined.

(no-undef)